docs/DEVELOPER-GUIDE.md

```diff
@@ -401,6 +448,34 @@ Add compatibility blocks to calendar JSON files:
 }
 ```
 
+### Cache Management
+
+#### Development Mode
+
+Season & Stars automatically detects development mode and skips caching for better iteration:
+
+```javascript
+// Cache is automatically skipped in development
+const isDevMode = game.data.world.flags['seasons-and-stars']?.devMode;
+// or when URL contains 'foundrytest'
+const isTestMode = window.location.href.includes('foundrytest');
```

Comment:
Thinking on it more, maybe just look for a host named "localhost" and leave it at that. If they want to use a different hostname, they can manually enable dev mode.